packages/@okta/vuepress-site/docs/guides/identity-providers/index.md

```diff
@@ -16,9 +16,7 @@ Not sure what an Identity Provider is? See [External Identity Providers](/docs/c
   <Card href="/docs/guides/add-an-external-idp/openidconnect/main/" headerImage="/img/idp-logos/oidc.png">OpenID Connect</Card>
   <Card href="/docs/guides/add-an-external-idp/oktatookta/main/" headerImage="/img/idp-logos/okta.svg">Okta to Okta</Card>
   <Card href="/docs/guides/add-an-external-idp/azure/main/" headerImage="/img/idp-logos/azure.svg">Azure AD</Card>
-  <Card href="/docs/guides/add-an-external-idp/smartcard/" headerImage="/img/idp-logos/smart-card.png">Smart Card</Card>
-  <!-- Need to figure out a way to link externally -->
-  <!-- <Card href="https://help.okta.com/okta_help.htm?id=ext-idp-smart-card-expression" headerImage="/img/idp-logos/smart-card.png">Smart Card</Card> -->
+  <Card href="https://help.okta.com/okta_help.htm?id=ext-idp-smart-card-expression" headerImage="/img/idp-logos/smart-card.png">Smart Card</Card>
```

Comment:
@vanngo-okta right now the link for SmartCard will open in a new tab/window as the default is to do so whenever the link is not internal. If you wish to force it to stay in the same tab since it's still an Okta URL, we can talk through further changes you'd need to make.

Reply:
The desired behaviour is to open a new tab/window for external links. This is consistent with external links for the whole site, so we are good 👍 .